community/bolt/src/docs/dev/examples.asciidoc

```diff
@@ -438,8 +438,8 @@ Server: SUCCESS {
 ----
 === Accessing notifications
 When Neo4j executes a statement it may include notifications for the user.
-These notifications can be warnings about problematic statements or other valuable information for a client. 
-Unlike failures or errors, notifications do not affect the execution of a statement.
+These notifications can be warnings about problematic statements or other valuable information for a client.
+Notifications are only included when using `EXPLAIN`.
```

Comment:
It is a pretty hard fact, it might change though. I just thought it might be confusing for someone trying to implement this without that piece of information.